src/components/UserInfo.js

```diff
@@ -1,18 +1,25 @@
 export default class UserInfo {
-  constructor(nameSelector, descriptionSelector) {
+  constructor(nameSelector, descriptionSelector, userAvatar) {
     this._nameElement = document.querySelector(nameSelector);
     this._descriptionElement = document.querySelector(descriptionSelector);
+    this._userAvatar = userAvatar
```

Comment:
You need to pass `userAvatarSelector` as well to search the avatar here to keep the code consistent

```js

    this._userAvatar = document.querySelector(userAvatarSelector);

```